components/table/Table.razor.cs

```diff
@@ -635,6 +636,23 @@ protected override void OnInitialized()
             FieldFilterTypeResolver ??= InjectedFieldFilterTypeResolver;
         }
 
+        public override async Task SetParametersAsync(ParameterView parameters)
+        {
+            if (parameters.TryGetValue<string>(nameof(ScrollBarWidth), out var value))
+            {
+                if (value is null)
+                {
+                    var scrollBarSize = await JsInvokeAsync<double>(JSInteropConstants.DomMainpulationHelper.GetScrollBarSize, false);
```

Comment:
Please get the scrollbar size once and cache the value globally.

Reply:
added `ClientDimensionService.cs`, now we can get the scrollbar size once and cache the value globally in C#.